config/foreman.hiera/tuning.yaml

```diff
@@ -5,3 +5,11 @@ apache::mod::passenger::passenger_max_request_queue_size: 250
 apache::mod::passenger::passenger_stat_throttle_rate: 120
 apache::mod::passenger::passenger_max_requests: 10000
 apache::mod::passenger::passenger_min_instances: 1
+apache::mod::passenger::passengerMaxPreloaderIdleTime: 0
+apache::mod::passenger::passengerPassengerStartTimeout: 500
```

Comment:
We override this in puppet-foreman and stopped using it in puppet-puppet so does this add anything?